### DISCUSSION TITLE
Is there any way to get json data from excalidraw using javascript?

### DISCUSSION THREAD

**Discussion by GurdeepSingh55** [excalidraw/excalidraw#7013] [General]
I am trying to use excalidraw in mvc project. I want to get all the scene elements using javascript. Is there any way to get json data from excalidraw using javascript?

**Comment by ghost** [comment]
<img width="506" alt="image" src="https://github.com/excalidraw/excalidraw/assets/6542274/d7a3b302-a742-4764-a59b-924e8fe768a3">
call that method from javascript

**Comment by JayZhouzzj** [comment]
The current drawing data is stored under localStorage of the browser. You can use browser inspect tool to see.   In js you can get it by localStorage.getItem("excalidraw");
